Assignment_0/200759_rahul.md

```diff
@@ -0,0 +1,55 @@
+# **Assignment 0**
+**Topic**
+1. [Commands](#Commands)
+2. [Vim](#Vim)
+3. [Git](#Git)
+4. [Github](#github)
+>## *Commands*
+`whoami` --> who you are logged in as <br>
+`ls` --> directry listing <br>
+`mv file1 file 2` --> move file 1 to file 2<br>
+`sudo apt install ..package name..` --> for installation of any package <br>
+`cd dir` --> change directory to dir<br>
+`cd ..` --> change to home<br>
+`sudo -s` --> open a root shell<br>
+`ifconfig` --> show network information<br>
+> ## *Vim*
+`:q` --> quit<br>
+`:w .filename` --> save file<br>
+`:q!` --> quit whitout saving<br>
+`:wq` --> save and quit <br>
+`zz` --> save and quit <br>
+`i` --> insert mode <br>
+`o` --> next line <br>
+`O` -->  previous line<br>
+`u` --> undo <br>
+`C,R` --> Redo<br>
+`o` --> jump to start of line<br>
+`$`-->jump to end of line <br>
+`gg` --> jump into first line<br>
+`yy` --> copy <br>
+`p` --> paste <br>
+`shift + v` --> visual mode <br >
+> ## *Git*
```

Comment:
Remove `>` and add newlines above and below a heading. Writing a proper README/md file is critical.